ecosystem/sep-0006.md

```diff
@@ -1121,6 +1121,8 @@ Name | Type | Description
 `refunds` | object | (optional) An object describing any on or off-chain refund associated with this transaction. The schema for this object is defined in the [Refunds Object Schema](#refunds-object-schema) section below.
 `required_info_message` | string | (optional) A human-readable message indicating any errors that require updated information from the user.
 `required_info_updates` | object | (optional) A set of fields that require update from the user described in the same format as [/info](#info).  This field is only relevant when `status` is `pending_transaction_info_update`.
+`customer_info_type` | string | (optional) The [SEP-12 type](sep-0012.md#type-specification) of customer information that is required to complete the transaction. This field is only relevant when `status` is `pending_customer_info_update`. Wallets should use this field to determine which fields to request from the user.
```

Comment:
I think we should provide the set of SEP-9 fields required on the transaction rather than use a SEP-12 customer type, which is just adding a level of indirection. It also requires businesses to group and label every potential set of SEP-9 fields required, which might be tedious if there are a lot of permutations.